spec.html

```diff
@@ -14451,13 +14457,19 @@ <h1>Static Semantics: Early Errors</h1>
           It is a Syntax Error if |LeftHandSideExpression| is either an |ObjectLiteral| or an |ArrayLiteral| and |LeftHandSideExpression| is not covering an |AssignmentPattern|.
         </li>
         <li>
-          It is an early Reference Error if |LeftHandSideExpression| is neither an |ObjectLiteral| nor an |ArrayLiteral| and IsValidSimpleAssignmentTarget of |LeftHandSideExpression| is *false*.
+          It is an early Reference Error if |LeftHandSideExpression| is neither an |ObjectLiteral| nor an |ArrayLiteral| and IsValidSimpleAssignmentTarget of |LeftHandSideExpression| is ~invalid~.
```

Comment:
It seems strange to have something named as a predicate (IsValidSimpleAssignmentTarget) not be a boolean. Should this be renamed to something else?

Reply:
`SimpleAssignmentTarget`, `AssignmentTarget{Type,Hint}` should do the trick or whatever @anba decides works for me as well.

For anything else, this PR LGTM. +1

Reply:
AssignTargetType: +1